zcash_client_sqlite/src/wallet/transparent.rs

```diff
@@ -421,7 +469,10 @@ pub(crate) fn add_transparent_account_balances(
         account_balances
             .entry(account)
             .or_insert(AccountBalance::ZERO)
-            .add_unshielded_value(value)?;
+            .with_unshielded_balance_mut::<_, SqliteClientError>(|bal| {
+                bal.add_pending_spendable_value(value)?;
+                Ok(())
+            })?;
```

Comment:
```suggestion
            .with_unshielded_balance_mut(|bal| bal.add_pending_spendable_value(value))?;
```
This is equivalent (compile-tested and rustfmt'ed).